### PR TITLE
Fix issues with version lock caching in version instances

### PR DESCRIPTION
**Pull request by chaosk**

### CODE REVIEW

**Comment by Aiky30** [comment]
@chaosk I'll replicate the issue locally and then check this branch out to test that this fixes it.

**Comment by Aiky30** [comment]
@chaosk The issue still exists. The only way I get roadbuster to work again is to rewind versioning to versioning 0.11

**Comment by monikasulik** [comment]
@chaosk @Aiky30 Is it possible to write a test that reproduces this issue reliably?

**Comment by chaosk** [comment]
@monikasulik I couldn't reproduce it locally, so I'm not sure what the test would be. 
@Aiky30 I've used a different approach now, I've deployed this to gaps-test and it seems I'm able to create a page there. Can you retest?

File: djangocms_version_locking/helpers.py [hunk 1]
```diff
@@ -87,22 +87,22 @@ def placeholder_content_is_unlocked_for_user(placeholder, user):
 
 def create_version_lock(version, user):
```
Comment (Aiky30) [inline djangocms_version_locking/helpers.py]: Should this also be renamed to reflect that the helper is getting or creating? i.e. get_or_create_version_lock

**Pull request merged**